module/zfs/abd.c

```diff
@@ -1408,10 +1408,13 @@ abd_raidz_gen_iterate(abd_t **cabds, abd_t *dabd,
 		switch (parity) {
 			case 3:
 				len = MIN(caiters[2].iter_mapsize, len);
+				break;
 			case 2:
 				len = MIN(caiters[1].iter_mapsize, len);
+				break;
 			case 1:
 				len = MIN(caiters[0].iter_mapsize, len);
+				break;
```

Comment:
It's my understanding that this fallt-hru is intentional so `len` will be the minimum of all valid `caiters[i].iter_mapsize`'s.  The same is true for `abd_raidz_rec_iterate`.